src/languages/es.ts

```diff
@@ -674,6 +674,7 @@ export default {
         settledExpensify: 'Pagado',
         settledElsewhere: 'Pagado de otra forma',
         individual: 'Individual',
+        business: 'Empresa',
```

Comment:
I've asked about translation confirmation in the [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1718281196269319)